services/web/server/src/simcore_service_webserver/exporter/formatters/sds/text_files.py

```diff
@@ -1,5 +1,5 @@
-from typing import Union
 from pathlib import Path
+from typing import Union
```

Comment:
Oh boy, those are really a lot of formatting in exporter ... :-)

Could you at some point also address [the bugs flagged](https://sonarcloud.io/project/issues?resolved=false&types=BUG&id=ITISFoundation_osparc-simcore) in this submodule  as well in a separate PR?

Reply:
We have a CIMIS case at some point and will have to go over it. I'll let it for that time.